app/views/browse/stateful-set.html

```diff
@@ -148,26 +148,24 @@
                 <uib-tab active="selectedTab.environment" ng-if="statefulSet">
                   <uib-tab-heading>Environment</uib-tab-heading>
                   <div class="resource-environment">
-                    <ng-form name="forms.statefulSetEnvVars">
```

Comment:
Looks like it. I have a separate WIP PR now for stateful sets (#1815). Will remove stateful sets changes from this one.